data/101/784/551/101784551.geojson

```diff
@@ -51,11 +51,11 @@
         "Ried im Innkreis"
     ],
     "name:eng_x_preferred":[
-        "Reid"
+        "Ried im Innkreis"
```

Comment:
This looks like a disambiguation name to me.

What about `Ried` as English prefered name, and `Ried im Innkreis` as name variant &/or disambiguation name instead? WOF default name should also update to `Ried`?

Reply:
> WOF default name should also update to Ried?

Hmm maybe I don't understand the issue here, but this place record represents [this Reid in Innkreis](https://en.wikipedia.org/wiki/Ried_im_Innkreis). The current `wof:name` value is correct here, we just need to swap the variant and preferred English names here..

Reply:
When visiting the [town website](https://www.ried.at/) they self-identify as `Ried`.

In Germany the practice is to add a disambiguation term after the city name when the city name is too common in GIS data and tabular data so they become more unique. In this case the string is saying "the city on the moist soil in the area of Innkreis". 

When we've cleaned other localities in Germany we've preferred to keep the basic city name as the wof name, and store the disambiguation name in that property instead.

See also:

- [{lang}_x_preferred_disambiguation](https://github.com/whosonfirst/whosonfirst-properties/blob/78d4adeacb2dd27cc8d0f74b139081ea5eca3972/properties/label/README.md#lang_x_preferred_disambiguation)

That said we're not super strict about it: https://spelunker.whosonfirst.org/id/101913837/

Reply:
Ah that makes sense.. and I'm remembering this issue coming up when admin data in Germany was last updated.

I agree with your proposal.. since this PR was filed through the writefield tool/on a fork, I'm going to close this out and open a new PR in its place.

Reply:
Replaced with https://github.com/whosonfirst-data/whosonfirst-data-admin-at/pull/17.